### ISSUE TITLE
Connection to a remote DATABASE (Error: unavailable database)

### ISSUE THREAD

**Issue by machkonti** [hgourvest/node-firebird#179]
Hello, I'm trying to connecto to remote database without any success.
The remote database is located on 192.xxx.xx.x:E\database\data.fdb

var fb          = require('node-firebird');
var options = {
    host: '192.xxx.xx.x',
    port: '3050',
    database: '192.xxx.xx.x:E:/database/data.fdb',
    user: 'SYSDBA',
    password: 'masterkey',
    loowecase_keys: false,
    role: null,
    pageSize: 4096
};
fb.attach(options, (err, db) => {
    if(err) {
        console.log(err);
        throw err;
    }
    db.query('SELECT * FROM table', (err, result) => {
        console.log(result);
        db.detach();
    });
}); 

whit this or escaped backslashes i recieve : `Error: unavailable database`
I have tryed many combinations of ip and path and all a wrong.
Is there any chance this can happend ?

**Comment by machkonti** [comment]
fixed:
database: 'E:\\database\\data.fdb',
